vaadinfrontend/src/main/java/life/qbic/datamanager/views/AppRoutes.java

```diff
@@ -1,24 +1,44 @@
 package life.qbic.datamanager.views;
 
 /**
- * Defines the data manager's public application routes, that
- * shall be visible by all components within the app.
- *
+ * Defines the data manager's public application routes, that shall be visible by all components
+ * within the app.
+ * <p>
  * Local routes of components should be listed here with care!
  *
  * @since 1.0.0
  */
 public class AppRoutes {
```

Comment:
What is the reason for not defining the AppRoutes as an Enum?

Reply:
Why enums? we need the character sequence representation for the route annotation value property

Reply:
Also more of a NIT but: 
The purpose of an enum is to make sure only the defined values are to be used, which makes sense in this context since only these values will provide sensible routing. 
Additionally by combining the Enum Constant with a String value similiar to [previous implentations](https://github.com/qbicsoftware/data-model-lib/blob/5ddc9812bb2445d3b22f1bbf985fc90e40f5fd18/src/main/groovy/life/qbic/datamodel/dtos/business/AcademicTitle.groovy) you'd still get the same result.